articles/fin-ops-core/dev-itpro/deployment/arm-onbarding-us-goverment.md

```diff
@@ -24,7 +24,10 @@ To deploy Azure Resource Manager topologies, you must complete the Resource Mana
 - The Azure subscription ID that you're deploying to
 
     > [!NOTE]
-    > For US government Microsoft Dynamics Lifecycle Services (LCS) projects, only Azure US government–specific Azure subscriptions are supported.
+    > For US government Microsoft Dynamics Lifecycle Services (LCS) projects, 
+    > 1. Only Azure US government–specific Azure subscriptions are supported.
+    > 2. Azure Subscriptions where management certificate support is not available, for example CSP Azure subscriptions are **not** supported currently.
```

Comment:
The term used to refer to the government subscriptions is "Azure Government" so I made that change below.
See https://learn.microsoft.com/en-us/azure/azure-government/documentation-government-welcome

This article is specific to Azure Government so I don't think we need bullet #1 because it is assumed. That means the new line could be the entire note with a couple of edits added.

Note
Azure Government Azure subscriptions where management certificate support is not available, for example Azure subscriptions purchased through a Cloud Service Provider or CSP, are **not** supported.

Reply:
We need to keep #1 because customers keep trying to use their commercial/public subscriptions here and then when it doesn't work then create support requests, this is why this line was added initially. 
I have pushed a change to use Azure Government subscription.

Reply:
@saurabhsurana Are further changes needed, or do I review and publish as is? Is there a work item associated with this change? If so, please assign it to me. Thanks!